core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -746,6 +746,8 @@ public enum Property {
       "The number of threads used to delete RFiles and write-ahead logs", "1.3.5"),
   GC_TRASH_IGNORE("gc.trash.ignore", "false", PropertyType.BOOLEAN,
       "Do not use the Trash, even if it is configured.", "1.5.0"),
+  GC_TRASH_IGNORE_IMPORTS_ONLY("gc.trash.ignore.imports.only", "false", PropertyType.BOOLEAN,
+      "Skip Trash for Import files when the trash is in use.", "3.0.0"),
```

Comment:
In spite of the description, it's not intuitive how this new property will interact with the previous `gc.trash.ignore` property. Rather than have a new BOOLEAN that modifies the behavior of the previous BOOLEAN only when the previous one was "true", how about changing `gc.trash.ignore` to accept a third possible value?

Reply:
@ctubbsii,  I was actually thinking you were going to take the discussion in a different direction and also suggest that we  might consider doing this on a per table basis instead of as an all or nothing approach (all tables or no tables) as is now.  That it would be better to make this easier and less error prone for users instead of having them manage and configure something like regexs over Accumulo files in HDFS trash.  From a durability standpoint, we allow walogs and replication to be configured on a per table basis.  Trash is just another safety net that a user may wish to user.  Some tables can be more important than others and allowing that decision to be made on a per table basis to include bulk input files, seems like a useful feature.  

I don't see this as Accumulo managing the trash.  I see this as more of giving the user explicit control over what Accumulo puts in the trash.    Once it is in the trash Accumulo isn't doing anything with it.

Reply:
Oh, I like the suggestion of one setting.  Something like gc_skip_trash = { ALL_FILES, BULK_FILES, NO_FILES}.  Is that more inline with what you were thinking?

Reply:
Yes, that's what I was thinking when I said "third possible value" above. But, that solution is only good if that is the end of the requirements to expand this behavior and the number of configurable values stops at three. If the requirements expand to support additional use cases, then it would be better to make this behavior a pluggable policy than to enumerate a long list of possible values.